src/main/java/de/bonndan/nivio/input/external/github/GitHubRepoHandler.java

```diff
@@ -0,0 +1,75 @@
+package de.bonndan.nivio.input.external.github;
+
+import de.bonndan.nivio.input.dto.ComponentDescription;
+import de.bonndan.nivio.input.dto.ItemDescription;
+import de.bonndan.nivio.input.external.ExternalLinkHandler;
+import de.bonndan.nivio.model.Link;
+import org.kohsuke.github.GHIssueState;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.GitHub;
+import org.kohsuke.github.MarkdownMode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.lang.NonNull;
+import org.springframework.util.StringUtils;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * https://github-api.kohsuke.org/index.html
+ */
+public class GitHubRepoHandler implements ExternalLinkHandler {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(GitHubRepoHandler.class);
+
+    public static final String OPEN_ISSUES = "github.issues.open";
+    public static final String OPEN_PRS = "github.pullrequests.open";
+    public static final String README = "github.readme";
+    public static final String DESCRIPTION = "github.description";
+
+    private final GitHub gitHub;
+
+    public GitHubRepoHandler(GitHub gitHub) {
+        this.gitHub = gitHub;
+    }
+
+    /**
+     * Loads the open issues from the repo.
```

Comment:
This is cool 😻 

Shouldn't the description though be something like "Loads the open issues, open pull requests, the readme and the repositories description"?

Reply:
done